tests/basic.txt

```diff
@@ -1,13 +1,15 @@
-Hello, World!
-
-This is some e-mail content. Even though it has whitespace and newlines, the e-mail converter will handle it correctly.
-
-Even mismatched tags.
-A div
-Another div
-A div
-within a div
-
-Another line
-Yet another line
+Hello, World!
```

Comment:
Newline in front of the link at the end. It was missing before. And yeah, the newline format got changed because I just overwrote the old file, but that's irrelevant.